### PR TITLE
Add campaign manager module

### PR DESCRIPTION
**Pull request by dimasciput**
Fix #117 
I'm using Blueprint functionality from Flask to create campaign_manager module

If we go to '/' then it will open reporter module, and if we go to '/campaign_manager' then it will go to campaign_manager module.

### CODE REVIEW

File: migrations/env.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
```
Comment (cchristelis) [inline migrations/env.py]: Why are we using SQLALCHEMY?

Comment (dimasciput) [inline migrations/env.py]: This migrations file is not used now, when we want to use the database then we can use these files. Should I just delete these too?

Comment (dimasciput) [inline migrations/env.py]: I have disabled the database integrations.

**Comment by Gustry** [comment]
This has been moved to its own repo https://github.com/hotosm/field-campaigner

**Pull request closed without merging**